src/color.rs

```diff
@@ -475,7 +471,7 @@ fn parse_rgb_components_rgb<'i, 't>(arguments: &mut Parser<'i, 't>) -> Result<(u
         }
         Token::Percentage { unit_value, .. } => {
             red = clamp_unit_f32(unit_value);
-            green = clamp_unit_f32(match arguments.next()? {
+            green = clamp_unit_f32(match arguments.next()?.clone() {
```

Comment:
Parsing initially became slower when `Parser::next` would clone every single token in order to put in the cache. Returning `&Token` removed these clones. In a minority of cases like here, the callers had to do these clones themselves instead. Note though that `Token::clone` never allocates. (Refcounting is used when a token contains an allocation.)

I don’t know the reason for the more recent regression. I’ll investigate next week.